src/common/components/Module.jsx

```diff
@@ -49,7 +49,7 @@ function Module({
         title={t('modules.target-blank-title')}
         isReadonly
         description={t('modules.target-blank-msg', { title: data.title })}
-        link={data.url}
+        link={`${data.url}?lang=${lang}`}
```

Comment:
Don't we have a function to add querystrings to a URL? This function should check if the URL already has a querystring added in order to add the new one with `&` instead of `?`